configapi/api_sub_config.go

```diff
@@ -392,6 +392,18 @@ func PostSubscriberByID(c *gin.Context) {
 
 	logger.WebUILog.Infoln("Received Post Subscriber Data from Roc/Simapp: ", ueId)
 
+	// Check if the IMSI already exists in the database
+	filter := bson.M{"ueId": ueId}
+	subscriber, err := dbadapter.CommonDBClient.RestfulAPIGetOne(amDataColl, filter)
+	if err != nil {
+		logger.DbLog.Errorw("failed to check subscriber existence", "IMSI", ueId, "error", err)
+		c.JSON(http.StatusInternalServerError, gin.H{"error": fmt.Sprintf("failed to check subscriber existence: %v", err)})
+		return
+	} else if subscriber != nil {
+		logger.WebUILog.Errorf("subscriber with IMSI %s already exists", ueId)
+		c.JSON(http.StatusConflict, gin.H{"error": fmt.Sprintf("subscriber with IMSI %s already exists", ueId)})
```

Comment:
```suggestion
		logger.WebUILog.Errorf("subscriber %s already exists", ueId)
		c.JSON(http.StatusConflict, gin.H{"error": fmt.Sprintf("subscriber %s already exists", ueId)})
```
the ueId is `imsi-XXXX` so it seems redundant, but this is just personal preference

Reply:
OK, removed.